rclcpp/test/rclcpp/node_interfaces/test_node_parameters.cpp

```diff
@@ -95,15 +95,15 @@ TEST_F(TestNodeParameters, list_parameters)
     std::find(list_result2.names.begin(), list_result2.names.end(), parameter_name),
     list_result2.names.end());
 
-  // Check prefixes
+  // Check prefixes and the depth relative to the given prefixes
   const std::string parameter_name2 = "prefix.new_parameter";
   const rclcpp::ParameterValue value2(true);
   const rcl_interfaces::msg::ParameterDescriptor descriptor2;
   const auto added_parameter_value2 =
     node_parameters->declare_parameter(parameter_name2, value2, descriptor2, false);
   EXPECT_EQ(value.get<bool>(), added_parameter_value.get<bool>());
```

Comment:
sorry i missed it.

```suggestion
  EXPECT_EQ(value2.get<bool>(), added_parameter_value2.get<bool>());
```

Reply:
Oh, I didn't check it. it looks like there was a mistake in the original code.
fixed. :thumbsup: